modules/git/ref.go

```diff
@@ -108,5 +137,25 @@ func (ref RefName) RefGroup() string {
 	if strings.HasPrefix(refName, PullPrefix) && strings.IndexByte(refName[pullLen:], '/') > -1 {
 		return "pull"
 	}
+	if strings.HasPrefix(refName, ForPrefix) {
+		return "for"
+	}
 	return ""
 }
+
+// RefURL returns the absolute URL for a ref in a repository
+func RefURL(repoURL, ref string) string {
+	refFullName := RefName(ref)
+	refName := util.PathEscapeSegments(refFullName.ShortName())
```

Comment:
Considering that there is such code for ShortName

```
	if strings.HasPrefix(refName, ForPrefix) {
		return strings.TrimPrefix(refName, ForPrefix)
	}
```


Is there the call to ShortName right? Especally the `assume they mean a branch` part